server/src/main/kotlin/tech/figure/objectstore/gateway/repository/DataStorageAccountsRepository.kt

```diff
@@ -6,10 +6,13 @@ import tech.figure.objectstore.gateway.model.DataStorageAccount
 
 @Repository
 class DataStorageAccountsRepository {
-    fun addDataStorageAccount(accountAddress: String): DataStorageAccount = transaction {
-        DataStorageAccount.new(accountAddress)
+    fun addDataStorageAccount(accountAddress: String, enabled: Boolean = true): DataStorageAccount = transaction {
```

Comment:
Not super useful for anything but tests, but there's no reason we can't include this flag